src/sentry/incidents/logic.py

```diff
@@ -971,12 +971,11 @@ def get_triggers_for_alert_rule(alert_rule):
 def trigger_incident_triggers(incident):
     from sentry.incidents.tasks import handle_trigger_action
 
```

Comment:
Similar adjustment here as in `handle_trigger_actions`, we're now sending the resolved message for all the alert rule trigger actions instead of just the trigger's actions. This function is only used when an incident is manually resolved.